### ISSUE TITLE
Casing of channel type keywords in channels.tsv needs clarification

### ISSUE THREAD

**Issue by sappelhoff** [bids-standard/bids-specification#534]
crossref:
- https://github.com/bids-standard/bids-validator/issues/1018
- https://github.com/mne-tools/mne-bids/issues/481

for the `type` column in `channels.tsv` files, we accept keywords from a restricted list.

In the BIDS specification, these keywords are all upper case: E.g., `EEG`, `RESP`, ... yet we do not specify that users should follow that casing.

IMO we should add a note to the spec that the casing MUST be as it is in the list in the specification.

cc @jasmainak @hoechenberger @agramfort

**Comment by hoechenberger** [comment]
It would be interesting to have some stats on how widespread the use of non-uppercase `type`s already is…

**Comment by jasmainak** [comment]
this reminds me of this dataset: https://github.com/bids-standard/bids-validator/issues/804. Seems like this needs to be clarified specification-wide

**Comment by dorahermes** [comment]
This is an error in 2 of the iEEG bids-examples, shall I correct these via pull request?

**Comment by sappelhoff** [comment]
> This is an error in 2 of the iEEG bids-examples, shall I correct these via pull request?

Let's discuss first, whether we all agree that upper case channel types are REQUIRED and that it has always been that way.

For me that's crystal clear, because if I tell somebody "hey, this is a list of stuff you can use", I expect them to take stuff from that list, not make up their own thing that looks similar to the stuff from the list.

But opinions differ and I agree that we can make it even more clear that upper casing is REQUIRED.

Now if some people say "no" ... or if we find out that a substantial amount of datasets has upper and lower case mixed, that might (or might not) be an argument to rather say: upper or lower case doesn't matter, it's both fine.

(in the latter case we wouldn't have to correct datasets, so I suggest to wait a bit @dorahermes )

**Comment by jasmainak** [comment]
Note that in the participants file, we allow both cases: https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file. I think we need to somehow have a clarification that casing is important unless allowed explicitly.

**Comment by agramfort** [comment]
I would suggest to impose the uppercase

**Comment by guiomar** [comment]
I'd also go for the uppercase

**Comment by dorahermes** [comment]
+1 on uppercase

**Comment by sappelhoff** [comment]
I myself are +1 for uppercase only as well. And thanks to @adam2392 we have a PR ready at bids-validator to enforce this behavior. https://github.com/bids-standard/bids-validator/pull/1022

Does somebody want to make a case for allowing both lower and upper casing? --> or e.g., warning if it's not upper case ... but accepting (this is the same as RECOMMENDING upper case, instead of REQUIRING)

**Comment by adam2392** [comment]
tbh I'm for upper-case just makes life easier, but I can see why someone might want lower case to only warn, since it wasn't explicitly clear.

tldr: +1 for upper-case.

**Comment by sappelhoff** [comment]
I would say that a PR is welcome to clarify the specification that Upper case is REQUIRED! Volunteers wanted :-)

I would consider this a clarification rather than increasing a requirement level, because from the start we said to take a valid keyword from that list ... and that list was always uppercase.

Afterwards we can adjust @adam2392's PR  bids-standard/bids-validator#1022 and make sure the validator checks everything.
